### PR TITLE
Add test command

### PR DESCRIPTION
**Pull request by NetoDevel**
to run all tests:

`quarkus-cli test`

### CODE REVIEW

**Comment by machi1990** [comment]
@NetoDevel Hi, good to see you here too. Thanks for the addition, I'll check this later on and come with the feedback.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "quarkus-cli",
-  "version": "1.0.0-alpha-01",
+  "version": "1.1.0-alpha-01",
```
Comment (machi1990) [inline package.json]:
```suggestion
  "version": "1.0.0-alpha-02",
```

File: src/services/launch-test-service.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const buildMavenProps = require('../utils/build-maven-props');
+
+module.exports = args => {
+  const mvn = require('maven').create({});
+  return mvn.execute(['clean', 'install'], buildMavenProps(args)).catch(() => {});
```
Comment (machi1990) [inline src/services/launch-test-service.js]:
```suggestion
  return mvn.execute(['clean', 'test'], buildMavenProps(args)).catch(() => {});
```

File: src/commands/launch-test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const launchTestService = require('../services/launch-test-service');
+
+module.exports = program => {
+  program
+    .command('test')
+    .allowUnknownOption()
+    .description(`Run all tests of your application.\n`)
```
Comment (machi1990) [inline src/commands/launch-test.js]: Maybe we should make it it clear that we are running `jvm` test only and not native tests.

File: README.md [hunk 4]
```diff
@@ -35,11 +35,14 @@ Type `quarkus-cli config -h` for more help
 - To run the current project in dev mode use `quarkus-cli dev`
 - To run the current project in remote dev mode use `quarkus-cli remote-dev`
 
+## test
+- To run all test current project use `quarkus-cli test`
```
Comment (machi1990) [inline README.md]:
```suggestion
- To run all none native test  of the current project use `quarkus-cli test`
```

**Comment by machi1990** [comment]
I also noticed that tests are failing, I'll look at it.

File: src/commands/launch-test.js (hunk 3 above)
Comment (NetoDevel) [reply src/commands/launch-test.js]: Done.

File: README.md (hunk 4 above)
Comment (NetoDevel) [reply README.md]: Done.

File: src/services/launch-test-service.js (hunk 2 above)
Comment (NetoDevel) [reply src/services/launch-test-service.js]: Done.

File: package.json (hunk 1 above)
Comment (NetoDevel) [reply package.json]: Done.

**Comment by NetoDevel** [comment]
@machi1990, I managed to solve the problem in the tests.

**Comment by machi1990** [comment]
> @machi1990, I managed to solve the problem in the tests.

Thanks. I am going to update a few things for the commit history and delete an unused file.
Once that is done we can merge this patch.

**Comment by machi1990** [comment]
cherry-picked  into master. Thanks for this contribution.

**Pull request closed without merging**